web/src/app/models/userSocial.model.ts

```diff
@@ -0,0 +1,12 @@
+export interface UserSocial {
+  github: {
+    updatedAt: string,
+    followers: any,
+    following: any,
```

Comment:
Please do not use `any`, this will be `number`

Reply:
Noted